docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -18,7 +18,7 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 
```

Comment:
Change "Notify SDL that an event becomes active" to "Notify SDL that an event becomes active or inactive"